.github/workflows/deploy-server.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to GAE
+on:
+  workflow_run:
+    workflows: ["Lint, test and build"]
+    branches: [production]
+    types: [completed]
+
+env:
+  server_dir: mobile-browser-based-version/server
+  server_url: https://deai-313515.ew.r.appspot.com/
+
+jobs:
+  deploy:
+    name: Deploying to Google Cloud
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ env.server_dir }}
```

Comment:
This is only set for `run` steps, if you want to use it for `google-github-actions/deploy-appengine`, you can use its [`working_directory` input](https://github.com/marketplace/actions/deploy-to-app-engine#inputs)